### PR TITLE
Tamplate

### PR DESCRIPTION
**Pull request by siddhi22kakade**

### CODE REVIEW

File: src/Component/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      <p>you can now listen to millions of <br>
+      songs, audiobooks and podcasts</br>on any device
+      any where you like!</p>
+    </div>
+    <div class="power ">
+      <ul class="List">
+        <li class="icon">
+          <i class="fa-solid fa-music"></i>
+        </li>
+        <div class="Annual ">
```
Comment (vsvshinde1994) [inline src/Component/index.html]: div cannot be used as an immediate child of ul

File: src/Component/style.css [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    } 
+
+    .payment {
+        width: 80%;
+    }
+        /* .Type {
+            font: size 1px;;
+
+        } */
+body {
```
Comment (vsvshinde1994) [inline src/Component/style.css]: body tag css should be placed at the top

File: src/Component/style.css [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+.block {
+    background-color: white;
+    width: 90%;
+    max-width: 460px;
+    border-radius: 20px;
+    margin: 40px auto;
+    padding-bottom: 20px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+}
+.Type {
```
Comment (vsvshinde1994) [inline src/Component/style.css]: use same casing for classnames

File: src/Component/style.css [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    font-family: Arial, Helvetica, sans-serif;
+}
+.payment {
+    height: auto;
+    width: 90%;
+    max-width: 400px;  
+    background-color: blue;
+    margin: 30px auto 0 auto;
+    border-radius: 19px;
+    text-align: center;
```
Comment (vsvshinde1994) [inline src/Component/style.css]: create commong class for text align: center

File: src/Component/style.css [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+    max-width: 400px;  
+    margin: 20px auto; 
+}
+@media screen and (min-width: 1024px) {
+    .block {
+        width: 700px;
+    }
+}
+.power {
+    display: flex;
```
Comment (vsvshinde1994) [inline src/Component/style.css]: create common class for display: flex

File: src/Component/style.css [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    font-size: 1.4rem;
+  
+}
+.pass {
+    margin: 20px 20px;
+    color: gray;
+    text-align: center;
+    line-height: 1.5;
+    letter-spacing: 0.2px;
```
Comment (vsvshinde1994) [inline src/Component/style.css]: why letter spacing is needed?

File: src/Component/style.css [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+    }
+}
+.power {
+    display: flex;
+    justify-content: space-between; 
+    align-items: center; 
+    flex-wrap: wrap; 
+    padding: 10px;
+    width: 80%; 
+    box-sizing: border-box;
```
Comment (vsvshinde1994) [inline src/Component/style.css]: what is the use of border-box property??

File: src/Component/style.css [hunk 8]
```diff
@@ -0,0 +1,159 @@
+@media screen and (min-width: 768px) {
+    .block {
```
Comment (vsvshinde1994) [inline src/Component/style.css]: use meaningful class names

File: src/Component/index.html [hunk 9]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css">
```
Comment (vsvshinde1994) [inline src/Component/index.html:9]: are you using any fontawesome class for your assignment ? if on then remove this line .

File: src/Component/index.html [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+</head>
+<body>
+  <div class="block">
+    <div>
+      <img src="images/musiCartoonone.jpeg" class="img">
```
Comment (vsvshinde1994) [inline src/Component/index.html]: image src should be loaded using javascript

**Comment by siddhi22kakade** [comment]
Oky sir i'll make the changes and send the file part

**Pull request merged**